PPPC Utility.xcodeproj/project.pbxproj

```diff
@@ -454,18 +470,19 @@
 			buildSettings = {
 				BUNDLE_LOADER = "$(TEST_HOST)";
 				CLANG_ENABLE_MODULES = YES;
+				CODE_SIGN_IDENTITY = "Apple Development";
 				CODE_SIGN_STYLE = Automatic;
 				COMBINE_HIDPI_IMAGES = YES;
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = XPLDEEDNHE;
```

Comment:
Is this value what we want for the dev team?

Reply:
I'm not sure what is best to do here. We had left the default project as "unsigned" and that was working fine, but the unit tests are not able to run in that case. The error was not clear when running the unit tests, why it wasn't working, so having my dev account here will at least give people an error about code signing that most are familiar with and they can change it.

Reply:
Very strange. I tried setting both targets to None and Sign to Run Locally. It builds fine, and it looks like it tries to start the tests, but they never actually run. Just hangs there. 🤔 

![Screen Shot 2020-10-15 at 10 10 29 AM](https://user-images.githubusercontent.com/923876/96149328-cbca0b80-0ece-11eb-8a2b-2ce22f1dae41.png)

Reply:
Yep, that is what happens to me too. It would be nice if the tests ran or gave some sort of error to indicate we needed to turn on signing.

Reply:
If I remove the hardened runtime entitlement from the PPPC Util, then I can run the tests when using "sign to run locally".

Reply:
I got it to work by turning of the hardened runtime for debug builds. I think that seems like a good workaround.